sdk/trace/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -497,45 +497,4 @@ private Attributes getImmutableAttributes() {
     // otherwise, make a copy of the data into an immutable container.
     return attributes.immutableCopy();
   }
-
-  @Override
-  public String toString() {
```

Comment:
do we really want to get rid of this toString()?  We can fix the other one without getting rid of this one.

Reply:
I don't want users to call `toString` on the `Span` and parse the String to extract fields, they should do that only in the processor via the `ReadableSpan`.

Reply:
`toString` is important in debugging, e.g., it's presented well by the IDE. We should definitely not be removing `toString`'s we already have. While it'd be unfortunate if anyone parses the tostring, that's a bit too explicit foot-shooting to lose the debugging experience for normal cases.

Reply:
Agree with @anuraaga . Let's not get rid of valuable debugging information, even if a user might do something unsupported. They can also use reflection if they want to do bad things to themselves.

Reply:
I will move this to a separate PR and still fight it. I am not sure what debugging scenario you have in mind

Reply:
Reverted.